data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -322,6 +318,21 @@ def get_sandbox_dataset_id(self):
         """
         return self._sandbox_dataset_id
 
+    @abstractmethod
+    def get_affected_tables(self, *args, **keyword_args):
+        """
+        Method to get tables that will be modified by the cleaning rule.
+
+        :return: a list of table names that will be affected by this cleaning rule
+        """
+        pass
+
+    def set_affected_tables(self, value):
+        """
+        set the affected_tables for this class instance
+        """
+        pass
+
```

Comment:
what do you think about adding these as properties methods to the base class?  like
```
def get_affected_tables(self):
    return self.__affected_tables

def set_affected_tables(self, value):
    self.__affected_tables = value if value else []

affected_tables = property(get_affected_tables, set_affected_tables)
```
how do you feel about updating the base class properties the same way?